Gemfile

```diff
@@ -184,7 +184,7 @@ group :seed, :manageiq_default do
 end
 
 group :smartstate, :manageiq_default do
-  gem "manageiq-smartstate",            "~>0.2.18",       :require => false
+  gem "manageiq-smartstate", :require => false, :git => "https://github.com/aufi/manageiq-smartstate.git", :tag => "v0.2.18.1"
```

Comment:
Why don't you just require the new version of the GEM? Does it not work in the backport?

Reply:
Correct, there are conflicts related to  vmware_web_service (from 0.2.19), more details at https://bugzilla.redhat.com/show_bug.cgi?id=1720649#c2